### PR TITLE
Create way to add additional ca files

### PR DESCRIPTION
**Pull request by samshull**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,27 +10,38 @@ var trustedCa = [
   '/usr/local/etc/openssl/cert.pem'
 ];
 
-var caAll = https.globalAgent.options.ca || [];
+var caAll = https.globalAgent.options.ca = https.globalAgent.options.ca || [];
```
Comment (azweb76) [inline index.js:13]: I dont recall, but there was a reason I did not want to override https.globalAgent.options.ca. I believe if it's null or not set, it uses the internal trust list.

**Comment by azweb76** [comment]
Do you mind throwing in the use-case for this? Just curious. :)

**Comment by samshull** [comment]
I have a ca bundle file that is in a node package that is required for making requests to certain hosts and requiring people to install the file into a specific directory or adding it to their keychain is unnecessarily cumbersome.

**Comment by samshull** [comment]
I didn't realize the full implications of this change. Going a different route

**Pull request closed without merging**